docs/en/02_Developer_Guides/05_Extending/05_Injector.md

```diff
@@ -276,7 +276,8 @@ By default, services registered with Injector do not inherit from one another; T
 named services, which may not be actual classes, and thus should not behave as though they were.
 
 Thus if you want an object to have the injected dependencies of a service of another name, you must
-assign a reference to that service.
+assign a reference to that service. References are denoted by by using a percent and dollar sign, like in the 
```

Comment:
double 'by'

Reply:
D'oh! Thanks for pointing this out. I'll fix this now.